### PR TITLE
adds updating instructions to install.md

### LINKED ISSUE #4227: Document crio update using package managers
**Description**

(As requested in the slack channel) - create documentation how crio should be updated using package managers like yum and apt. Other projects like kubespray would need it.

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `crio --version`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

### PR DESCRIPTION
**Pull request by wgahnagl**
Signed-off-by: Skyler Clark <wgahnagl@protonmail.com>

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
adds updating instructions to the install docs
#### Which issue(s) this PR fixes:

fixes #4227 

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/ok-to-test
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
and happy green test buttons.

File: install.md [hunk 1]
```diff
@@ -513,3 +513,36 @@ sudo systemctl start crio
...
+```
+sudo dnf update
+sudo dnf update cri-o
+```
+
+### APT based operating systems
+be sure that the environment variable ```$OS``` is set to your operating system
+
+```
+rm /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
```
Comment (kolyshkin) [inline install.md:535]: You use `$VERSION` here before you explain what it should be set to.

File: install.md [hunk 2]
```diff
@@ -513,3 +513,36 @@ sudo systemctl start crio
...
+```
+
+### Fedora 31 or later
+```
+sudo dnf update
+sudo dnf update cri-o
+```
+
+### APT based operating systems
+be sure that the environment variable ```$OS``` is set to your operating system
```
Comment (kolyshkin) [inline install.md:532]: Would be nice to give an example here.

Comment (wgahnagl) [reply install.md]: does my update fix this or is it too much ?

**Comment by hafe** [comment]
I don't see instructions for centos and rhel

**Comment by hafe** [comment]
I think you should separate minor and patch upgrade. What is strange here is that when minor version change, a new repo file needs to be created

**Comment by umohnani8** [comment]
LGTM
@haircommander @kolyshkin PTAL
We can add instructions for Rhel and Centos in a final PR, would like to get this in first though.

**Comment by haircommander** [comment]
I agree with @hafe about the differences in version bumps. One should just do a `$PKG_MANAGER update` for patch versions (assuming they didn't pick a pinned version like 1.18.4), and totally update the files otherwise. I also think it's pretty incomplete without rpm based upgrade instructions

**Pull request merged**